apps/fluent-tester/src/TestComponents/Badge/BasicBadgeTest.tsx

```diff
@@ -68,11 +60,25 @@ export const BasicBadge: React.FunctionComponent = () => {
   const badgeConfig = {
     appearance: badgeAppearance,
     badgeColor,
-    position: 'absolute' as FlexStyle['position'],
     size,
     shape,
+    shadowToken: showShadow ? theme.shadows.shadow4 : undefined,
   };
 
+  const StyledBadge = Badge.customize({
```

Comment:
Oof I would really prefer this to stay outside of the render function and/or memoized in some way, because otherwise the StyledBadge component gets recreated on every render.

Reply:
I agree, but I can't use `theme` outside of render function. Because in this case I'll need to do `const theme = useFluentTheme();` on the top level and there will be an error: "React Hook "useFluentTheme" cannot be called at the top level.". I'll try to memoize it

Reply:
Using useMemo or creating a memoized function which takes in a theme should solve that issue.